15_mini_project_work/2_num_guessing_game.py

```diff
@@ -33,20 +36,26 @@ def main():
     print(GREETING)
 
     is_guess_wrong = True
+    rand_num = random.randint(RANDOM_LEFT_BORDER, RANDOM_RIGHT_BORDER)
```

Comment:
bad name

Reply:
Which one?

Reply:
![image](https://github.com/sergeroshe/Stepik_course/assets/62591932/30481b4b-cdb4-419d-81a2-dd5b57d50161)

Reply:
Still unclear. Which one/ones of const/var names is/are bad?